javascript-timers/interval.js

```diff
@@ -0,0 +1,25 @@
+var $display = document.querySelector('.countdown-display');
+var intervalID;
+
+function runFunction(number) {
+  intervalID = setInterval(changeDisplay, 1000);
+}
+
+function changeDisplay(number) {
+  if ($display.textContent === '4') {
+    $display.textContent = '3';
+
+  } else if ($display.textContent === '3') {
+    $display.textContent = '2';
+
+  } else if ($display.textContent === '2') {
+    $display.textContent = '1';
+
+  } else if ($display.textContent === '1') {
```

Comment:
What if the countdown was supposed to be from 1000? You should try to do this in a way where you don't need a conditional block for each number in the countdown.

Reply:
You're absolutely right. Just fixed. Thanks for the feedback, sir.